scripts/travis/test.sh

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+OS=$("${SCRIPTPATH}/../ostype.sh")
+ARCH=$("${SCRIPTPATH}/../archtype.sh")
+
+if [ "${OS}-${ARCH}" = "linux-arm" ]; then
+    # for arm, no tests need to be invoked.
+    exit 0
+fi
+
 make fixcheck
-if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then scripts/travis/run_tests.sh; fi
```

Comment:
why is the change needed? to unify with other scripts?

Reply:
I've just added a query for OS and PLATFORM few lines above this one. It would be weird to have the TRAVIS variables in one place and the OS/PLATFORM in another, so I modified them.